pages/wiki/creating-an-asteroid-app.md

```diff
@@ -163,6 +168,29 @@ If you want to disable screen locking for easier development you can enable the
 ```
 mcetool -D on
 ```
+
+# Building applications for the emulator
+The primary difference between the two versions of the SDK are that the real watches use an ARM processor, which the emulator uses a simulated x86 processor.  This allows the emulator to run much faster on computers based on the x86 architecture than if they had to emulate an ARM processor.
+
+## Prebuilt emulator SDK
+You can either download a prebuilt emulator SDK from the nightlies [here](https://release.asteroidos.org/nightlies/sdk/oecore-x86_64-core2-32-toolchain-nodistro.0.sh) and install it on your system or you can build it yourself in a process very similar to that described above for creating the watch version of the SDK.
+
+## Building and installng the emulator SDK
+The only difference between building the SDK for the emulator is that instead of using `./prepare-build.sh dory` one uses `./prepare-build qemux86`.  Installation is also largely the same except for a small difference in the command used to install.  Installing the emulator version of the SDK is done with this command:
+
+```
+./tmp-glibc/deploy/sdk/oecore-x86_64-core2-32-toolchain-nodistro.0.sh
+```
+
+## Using the emulator SDK
+As the emulator installer itself describes when it is first installed,
+
+>    Each time you wish to use the SDK in a new shell session, you need to source the environment setup script e.g.
+>     $ . /usr/local/oecore-x86_64/environment-setup-armv7vehf-neon-oe-linux-gnueabi
```

Comment:
If we're talking about the emulator SDK maybe you shouldn't source the arm version ?

Reply:
Good point.